requirements.txt

```diff
@@ -6,3 +6,4 @@ keyring>=10.4.0
 requests>=2.13.0,<3.0.0
 fido2>=0.7.0,<1.0.0
 okta>=0.0.4,<1.0.0
+aws-okta-keyman>=0.7.2
```

Comment:
Lets add an upper bound on the version here please (ie. `>=0.7.2,<1.0.0`).  Thanks!

Reply:
Or, if we pull in that Duo class, lets just remove this entry all together.

Reply:
Added upper bound to aws-okta-keyman version in requirements.txt